src/EA.Weee.Web/Areas/Producer/Controllers/ProducerController.cs

```diff
@@ -195,12 +195,13 @@ public async Task<ActionResult> RepresentedOrganisationDetails()
                 SmallProducerSubmissionData = SmallProducerSubmissionData
             };
 
+            var representingCompanyDetailsViewModel = mapper.Map<SmallProducerSubmissionMapperData, RepresentingCompanyDetailsViewModel>(source);
+
             var vm = new OrganisationDetailsTabsViewModel
             {
                 ActiveOption = OrganisationDetailsDisplayOption.RepresentedOrganisationDetails,
                 SmallProducerSubmissionData = this.SmallProducerSubmissionData,
-                RepresentingCompanyDetailsViewModel = SmallProducerSubmissionData.HasAuthorisedRepresentitive ? 
-                    mapper.Map<SmallProducerSubmissionMapperData, RepresentingCompanyDetailsViewModel>(source) : null
+                RepresentingCompanyDetailsViewModel = representingCompanyDetailsViewModel
```

Comment:
@stevelee-civica  looking at the changes now. Do we actually need this RepresentingCompanyDetailsViewModel on the viewmodel. We can access the rep company data through SmallProducerSubmissionData.AuthorisedRepresentitiveData etc...